DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Controllers/JobProfileSectionController.cs

```diff
@@ -110,7 +130,28 @@ private ActionResult ReturnSectionView()
         {
             var propertyValue = GetPropertyValue(CurrentJobProfile);
 
-            return View("Index", new JobProfileSectionViewModel { TopSectionContent = TopSectionContent, BottomSectionContent = BottomSectionContent, PropertyValue = propertyValue, Title = Title, PropertyName = PropertyName });
+            var model = new JobProfileSectionViewModel
+            {
+                TopSectionContent = TopSectionContent,
+                BottomSectionContent = BottomSectionContent,
+                PropertyValue = propertyValue,
+                Title = Title,
+                PropertyName = PropertyName,
+                IsWhatItTakesCadView = PropertyName.Equals(nameof(JobProfile.Skills), StringComparison.OrdinalIgnoreCase) && CurrentJobProfile.HowToBecomeData.IsHTBCaDReady
```

Comment:
This works i suppose. I was expecting a seperate widget for WIT with skills current as-is and the restrictions and other requirements based on the HTB CaD readiness, guess this works for now, and we had to change it as part of skills development. No action needed.